diff -up xmms-wma-1.0.5/xmms-wma.c.ff xmms-wma-1.0.5/xmms-wma.c
--- xmms-wma-1.0.5/xmms-wma.c.ff	2009-03-31 21:40:10.000000000 +0200
+++ xmms-wma-1.0.5/xmms-wma.c	2009-03-31 21:42:45.000000000 +0200
@@ -400,7 +400,8 @@ static void *wma_play_loop(void *arg)
         if(size == 0) break;
 	
         while(size > 0){
-            len = avcodec_decode_audio(c, (short *)wma_outbuf, &out_size,
+            out_size=sizeof(wma_outbuf);
+            len = avcodec_decode_audio2(c, (short *)wma_outbuf, &out_size,
                                        inbuf_ptr, size);
 	    if(len < 0) break;
 	    
